guide/docs/faq/general.mdx

```diff
@@ -50,10 +50,13 @@ bot = commands.Bot(..., activity=activity)
 
 ### How do I send a message to a specific channel?
 
-You must fetch the channel directly and then call the appropriate method. Example:
+You must get or fetch the channel directly and then call the appropriate method. Example:
 
 ```python
-channel = bot.get_channel(12324234183172)
+channel = bot.get_channel(12324234183172) or await bot.fetch_channel(12324234183172)
+# if Bot.get_channel returns None because the channel is not in the Bot's cache then it'll 
+# make an API call to fetch the channel
```

Comment:
```suggestion
# If Bot.get_channel returns None because the channel is not in the Bot's cache, 
# make an API call to fetch the channel
channel = bot.get_channel(12324234183172) or await bot.fetch_channel(12324234183172)
```
swapping the comment and code lines, plus tiny phrasing change to clarify the behavior